doc/userdoc/contribute/styleguide/styleguide.rst

```diff
@@ -0,0 +1,634 @@
+The NEST documentation style guide
+==================================
+
+.. contents:: On this page, you will find
+   :local:
+   :depth: 1
+
+Why do we have a style guide?
+-----------------------------
+
+This style guide was created to provide a single reference point for content
+creation in NEST. This helps ensure the NEST user-level documentation remains
+clear and consistent. The style choices we make here are meant to follow the
+major trends in technical writing for software.
+
+Contribute to the docs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to add or modify documentation, make sure to read through this guide before writing a contribution.
+
+We have templates for :doc:`Python example scripts <../templates/pyapi_template>` and the :doc:`PyNEST API docstrings <../templates/example_template>`.
+
+You will also need to know how to build the documentation locally on your machine: See
+:doc:`../../documentation_workflow/user_documentation_workflow`.
+
+For additional information, including the Git workflow and making a pull request see :doc:`../index`.
+
+
+A few tips on writing documentation
+-----------------------------------
+
+- Be concise: Use short words and sentences; avoid unnecessary modifiers.
+
+- Be consistent: Follow this style guide and relevant templates,
+
+- Be specific: Avoid vague language; clarity is key.
+
+- Focus your message: Start with what is most important, and keep text relevant
+  within given section.
+
+- Write for your reader: Try to understand what your reader needs to know;
+  include context to what you are saying. Remember that most NEST users are computational neuroscientists,
+  including students, not software developers!
+
+- Provide examples: Show, do not tell; if you can use code snippets, screenshots,
+  or figures to illustrate a point, do so!
+
+- Avoid terms like "simply", "just", or "easy": Your knowledge does not always equal
+  your readers.  Do not make your reader feel bad because they "simply" did not
+  understand something.
+
+- Write positively: Use positive language rather than negative language.
+
+Remember that the primary purpose of the documentation is to provide
+information to NEST users who are looking for it.
+
+The language we use
+-------------------
+
+We follow spelling and grammar rules of `American English <https://www.merriam-webster.com/>`_.
```

Comment:
From here on through a several of the following sections, we have single-sentence paragraphs, many quite short. Would it make more sense and be more pleasing to the reader's eye to turn this all into bullet lists?

Reply:
I updated all the single sentences so they are bullet lists, I think this looks better thanks